universum/modules/output/html_output.py

```diff
@@ -15,68 +15,57 @@ def __init__(self, *args, **kwargs):
         self._filename = None
         self.artifact_dir_ready = False
         self._log_buffer = list()
+        self._block_level = 0
 
     def set_artifact_dir(self, artifact_dir):
         self._filename = os.path.join(artifact_dir, "log.html")
 
     def open_block(self, num_str, name):
-        self.log(name) # stub
+        self._log_line(f"{num_str} {name}")
+        self._block_level += 1
 
     def close_block(self, num_str, name, status):
-        self.log(name) # stub
+        self._block_level -= 1
+        indent = "  " * self._block_level
+        self._log_line(f"{indent} \u2514 [{status}]")
+        self._log_line("")
 
     def report_error(self, description):
-        self.log(description) # stub
+        pass
```

Comment:
Is this behaviour going to be changed in further reviews?

Reply:
No, this behavior is copied from `TerminalBasedOutput`.